README.md

```diff
@@ -9,115 +8,14 @@ Blockchain DB is a **key-value/document database** with certain blockchain prope
-The installation steps for `jq` can be found [here](https://stedolan.github.io/jq/download/).
-
-For each `GET` request, we need to set two custom headers called `UserID` and `Signature` which will be included in the request while sending
-HTTP to the blockchain DB node.
-
-As every user who issues a `GET` or `POST` request must be authenticated cryptographically through digital signature, the submitting user must
-compute a signature on the query data or transaction data and set the signature in the `Signature` header. We use the [`signer`](./docs/curl/build.md#build-and-use-signer-utility) utility to
-compute the required digital signature.
-
-Queries can be used to fetch data stored/managed by the database. For example, we can perform the following:
-
-  1. [Querying the configuration](./docs/curl/query.md#querying-the-cluster-configuration)
-  2. [Querying a user information/credentials/privileges](./docs/curl/query.md#querying-the-user-information)
-  3. [Checking the existance of a database](./docs/curl/query.md#checking-the-database-existance)
-  4. [Querying data](./docs/curl/datatx.md#checking-the-existence-of-the-state)
-  5. [Querying provenance of data or user](./docs/curl/provenance.md)
-  6. [Querying a block](./docs/curl/query.md#querying-a-block-header)
-  7. [Querying a transaction receipt for async transactions](./docs/curl/ledger.md#transaction-receipt-query)
-  8. [Querying proof of existence of a transaction](./docs/curl/ledger.md#transaction-proof-query)
-  9. [Querying proof of existence of data](./docs/curl/ledger.md#state-proof-query)
-
-Error cases are discussed in [queries resulting in errors](./docs/curl/errors.md)
-
-### Transactions
-
-We support four types of transactions:
-
-  1. [Database administration transaction](docs/curl/dbtx.md)
-    - for creating and deleting databases.
-  2. [User administration transaction](docs/curl/usertx.md)
-    - for adding, deleting, updating users' credentials and privileges.
-  3. [Data transaction](docs/curl/datatx.md)
-    - for adding, deleting, updating the blockchain states, i.e., active key-value pairs.
-  4. [Cluser configuration transaction](docs/curl/configtx.md)
-    - for adding, deleting, updating nodes' and admins' configuration.
-
-
-### Proofs
-
-We support three kinds of proofs:
-- Proof of ledger connectivity and block existence in ledger.
-- Proof of transaction TxID been part of block N.
-- Proof of state - at the end of a specific block N the value of given key K was equal to X.
-
-Detailed explanation about these kinds of proofs can be found [here](docs/proofs/Proofs.md)
+  
+## Documentation 
+Please visit our online documentation for information on getting started [here](http://labs.hyperledger.org/orion-server/docs/introduction/)
 
 ## How to Contribute
-
-TODO
-
-## Community Mailing List & Rocket Chat
-
-TODO
+We don't have formal process how to become Orion contributor, so please contact project [maintainers](MAINTAINERS.md)
```

Comment:
We will be happy to receive help! In order to become an Orion contributor, please contact the project [maintainers](MAINTAINERS.md).